src/Main.java

```diff
@@ -1,5 +1,316 @@
 public class Main {
+    static Employee[] employees = new Employee[10];
+
+    public static void addEmployee(String lastName, String firstName, String middleName, int department, double salary) {
+        for (int i = 0; i < employees.length; i++) {
+            if (employees[i] == null) {
+                employees[i] = new Employee(firstName, middleName, lastName, department, salary);
+                break;
+            } else if (employees[i] == employees[employees.length - 1]) {
+                System.out.println("База данных работников переполнена.");
+            }
+        }
```

Comment:
```suggestion
        for (int i = 0; i < employees.length; i++) {
            if (employees[i] == null) {
                employees[i] = new Employee(firstName, middleName, lastName, department, salary);
                return;
            }
        }
        System.out.println("База данных работников переполнена.");

```
Можно ещё вот так написать